MD_Simulations_on_Local_Machine.md

```diff
@@ -0,0 +1,49 @@
+# Standard Operating Procedure:  
+## Running Molecular Dynamic Simulations on your Local Machine
+
+### 1. Purpose
+- This SOP outlines the steps necessary to run Molecular Dynamic Simulations using OpenMM and OpenFF on your local machine.
+
+### 2. Procedure
+##### 2.1 Downloading Necessary Software
+###### 2.1.1 Install Visual Studio Code
+- Follow the instructions to download VS Code [here](https://code.visualstudio.com/Download)
+###### 2.1.2 Install Miniconda
+- Follow the instructions to install Miniconda [here](https://www.anaconda.com/docs/getting-started/miniconda/install)
+###### 2.1.3 Install Julia
+- Follow the instructions to install Julia [here](https://julialang.org/downloads/)
+###### 2.1.4 Install Packmol
+- Follow the instructions to install Packmol [here](https://m3g.github.io/packmol/download.shtml)
+###### 2.1.5 Install OpenMM
+- Follow the instructions to install OpenMM [here](https://docs.openmm.org/latest/userguide/application/01_getting_started.html#introduction)
+###### 2.1.6 Install OpenFF
+- Follow the instructions to install OpenFF [here](https://docs.openforcefield.org/projects/toolkit/en/latest/installation.html)
+###### 2.1.7 Install VMD
+- Follow the instructions to install VMD [here](https://www.ks.uiuc.edu/Development/Download/download.cgi?PackageName=VMD)
```

Comment:
Did you have any issues installing any of these locally? Any tips for future ReACTERs?